### DISCUSSION TITLE
Welcome to OpenQuotes Discussions!

### DISCUSSION THREAD

**Discussion by MrAshwin2142** [MrAshwin2142/OpenQuotes#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JohnPatrickAncajas** [comment]
Hello! I'm Patrick and I'm currently training to be a full stack developer. This repo is not only web based but also beginner friendly so I'm currently here contributing and also learning!

**Reply by MrAshwin2142** [reply]
Hello Patrick , Ashwin this side and I'm also exploring frontend development, So create this repo to learn and help others to start their open source journey.

**Comment by MrAshwin2142** [comment]
🎉 Thank You, Awesome Contributors! 🙌

We wanted to express our heartfelt gratitude for your fantastic contributions to OpenQuotes. Your creativity, dedication, and support have made a significant impact.

As we continue our open source journey, we're excited to introduce you to our new project, 'The Wall of Projects.' It's a platform where you can explore, learn, and collaborate on a variety of web development projects. You're all invited to be part of this new adventure and continue expanding your web development skills.

Discover 'The Wall of Projects' at https://mrashwin2142.github.io/The-Wall-of-Projects/ and explore a world of exciting web development opportunities.

Once again, thank you for your contributions, and we hope to see you on 'The Wall of Projects' soon! 🚀🌈
@JohnPatrickAncajas 
@pheb2014939ctu 
@devraushan 
@VveksGit 
@Delta7Actual 
@diemmee
@HTQun 
@txlan1710
@YeiHuynh
